src/docs/release/breaking-changes/image-cache-and-provider.md

```diff
@@ -0,0 +1,140 @@
+---
+title: [Changes to ImageCache and ImageProvider]
+description: [ImageCache requires implementers to override containsKey, and
+ImageProvider has marked resolve as @nonVirtual]
+---
+
+
+## Summary
+
+`ImageCache` now has a method called `obaintsKey`. `ImageProvider` subclasses
+should not override `resolve`, but instead should resolve new methods on
```

Comment:
should resolve -> should implement 

or something like that?

Reply:
Oops - yeah.